pkg/model/dev.go

```diff
@@ -144,6 +144,7 @@ type Dev struct {
 	Resources            ResourceRequirements  `json:"resources,omitempty" yaml:"resources,omitempty"`
 	Services             []*Dev                `json:"services,omitempty" yaml:"services,omitempty"`
 	PersistentVolumeInfo *PersistentVolumeInfo `json:"persistentVolume,omitempty" yaml:"persistentVolume,omitempty"`
+	Autocreate           bool                  `json:"autocreate,omitempty" yaml:"autocreate,omitempty"`
```

Comment:
Is this the same as if the user passes the `deploy` arg in the CLI? If so, can we use the same name in the manifest to keep it consistent?

Reply:
`deploy` sounds confusing? what about `autodeploy`?
Also, we are keeping the `deploy` arg, but it should be deprecated

Reply:
I think calling the same thing two different ways is more confusing. People use the `--deploy` param today, it'll be confusing to them.

both are equally confusing to me TBH, the fact that we call it 'deploy'  or 'autocreate' is leaking that fact that we need a deployment before doing the `up` phase of the process. I'd rather stay with the current 'confusing' term rather than introduce yet another.

Reply:
@rberrelleza do you have numbers on how many are using `--deploy`? The idea is to deprecate the `deploy` arg

Reply:
not sure if we are collecting that information. 

maybe we can call it `mode: standalone`? (with the idea that then having `mode:swap` in the future so that users can be more explicit about how they want to use up?)

Reply:
if we are deprecating the deploy flag, maybe let's add a message to the help now?

Reply:
I thought that the idea of using a `mode` was interesting for a long time. We even discussed this several times in the past, but the truth that if we want to keep it simple that just adds a lot more complexity for a use case that is at its minimum:

According to the data: 

- 95.59% of swap use.
- 4.41% of the "standalone" use. Of which we already know many happened by mistake.

Having a simple flag called `autocreate: true` right now (or even `standalone: true`) allows us to keep this feature for that small percentage who use it and avoids the current mistake of creating an unwanted deployment for some. I don't think we should be blocked by the naming convention for this, as long as is not as confusing as `deploy` or `autodeploy`, which is a term we use for something completely different.

I vote for `autocreate: true` or `standalone: true`

Reply:
@jLopezbarb let's go with `autocreate` then